pkg/storaged/clevis-luks-passphrase.sh

```diff
@@ -21,7 +21,7 @@ if cryptsetup isLuks --type luks1 "$DEV"; then
     luksmeta show -d "$DEV" | while read slot state uuid; do
         if [ "$state" == "active" -a "$uuid" == "$CLEVIS_UUID" ]; then
             if pp=$(luksmeta load -d "$DEV" -s "$slot" | clevis decrypt); then
-                echo $pp
+                printf '%s\n' "$pp"
```

Comment:
I think you should then also change the hashbang to /bin/sh to indicate that. There are several test failures anyway, so you can slip that into the next push. Thanks!

Reply:
Ah, good point.

Reply:
Done, and fixed the rest as well.